src/styles/blocks/catalog.scss

```diff
@@ -11,15 +11,23 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    cursor: pointer;
 
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform 300ms ease, color 300ms ease;
```

Comment:
checklist p4. You can add transition duration as a variable since you use the same value for it more than once
Correct it in the whole doc

Reply:
Thanks! Just fixed it!